infra/azure/scripts/generate-env.js

```
@@ -99,9 +99,12 @@ const config = {
   AZURE_FUNCTIONS_BASE_URL: '',
   AZURE_APP_INSIGHTS_CONNECTION_STRING: '',
   AZURE_COSMOS_ENDPOINT: '',
-  AZURE_AD_B2C_TENANT_ID: '',
-  AZURE_AD_B2C_CLIENT_ID: '',
-  AZURE_AD_B2C_AUTHORITY: '',
+  AZURE_ENTRA_TENANT_ID: '',
```

ESLint

[error] Replace `''` with `""`

(prettier/prettier)